pkgs/desktops/plasma-5/kpipewire.nix

```
@@ -0,0 +1,15 @@
+{
+  mkDerivation,
+  extra-cmake-modules,
+  kwayland, ki18n, kcoreaddons, plasma-wayland-protocols,
+  libepoxy, ffmpeg, mesa, pipewire, wayland
+}:
```

```suggestion
{
, mkDerivation,
, extra-cmake-modules
, kwayland
, ki18n
, kcoreaddons
, plasma-wayland-protocols,
, libepoxy
, ffmpeg
, mesa
, pipewire
, wayland
}:
```
same as above and reflowing this makes the diffs always unnecessarily hard to understand.